v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/spanner/DatastreamToSpannerExceptionClassifierIT.java

```diff
@@ -101,6 +101,28 @@ public void testInterleaveInsertChildBeforeParent() {
     assertSpannerExceptionClassification(exception, RETRYABLE_ERROR, actualTag);
   }
 
+  @Test
+  public void testInterleaveInInsertChildBeforeParent() {
+    ErrorTag actualTag = null;
+    SpannerException exception = null;
+    Mutation mutation =
+        Mutation.newInsertBuilder("Series")
+            .set("id")
+            .to(4)
+            .set("author_id")
+            .to(100)
+            .set("title")
+            .to("Child")
+            .build();
+    try {
+      spannerResourceManager.writeInTransaction(List.of(mutation));
+    } catch (SpannerException e) {
+      exception = e;
+      actualTag = DatastreamToSpannerExceptionClassifier.classify(SpannerExceptionParser.parse(e));
+    }
+    Assert.assertNull(exception);
+  }
```

Comment:
`actualTag` should also be asserted right?

Reply:
actualTag is set in the exception flow. There is not exception

Reply:
That does not matter I think, we should be doing something like this -

https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/e0d274c3bd7f2eb4fb343daeb64775ddc668d6ee/v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/spanner/DatastreamToSpannerExceptionClassifierTest.java#L195


WDYT? This example seems to assert the output of the error classifier?